tools/scenario-player/scenario_player/tasks/execution.py

```diff
@@ -72,3 +72,15 @@ class WaitTask(Task):
 
     def _run(self, *args, **kwargs):  # pylint: disable=unused-argument
         gevent.sleep(self._config)
+
+
+class WaitBlocksTask(Task):
+    _name = 'wait_blocks'
+
+    def _run(self, *args, **kwargs):  # pylint: disable=unused-argument
+        web3 = self._runner.client.web3
+        start_block = web3.eth.blockNumber
+        end_block = start_block + self._config
```

Comment:
Using `int(self._config)` would be a bit more resilient in case someone writes `wait_blocks: "20"`